app/views/layouts/application.html.erb

```diff
@@ -1,14 +1,38 @@
-<!DOCTYPE html>
-<html>
+<!doctype html>
+<html lang="en">
   <head>
-    <title>Acebook</title>
-    <%= csrf_meta_tags %>
+      <%= csrf_meta_tags %>
 
     <%= stylesheet_link_tag    'application', media: 'all' %>
     <%= javascript_include_tag 'application' %>
-  </head>
+    <!-- Required meta tags -->
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+
+    <!-- Bootstrap CSS -->
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.0-beta2/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-BmbxuPwQa2lc/FVzBcNJ7UAyJxM6wuqIj61tLrc4wSX0szH/Ev+nYRRuWlolflfl" crossorigin="anonymous">
 
+     <title>Acebook</title>
+  </head>
   <body>
-    <%= yield %>
+  
+  <br/>
+  <%= render "welcome/header" %> <br>
+  <div class="container">
+     <%= yield %>
```

Comment:
This: `<div>` is not actually being closed off, could have potential conflicts later down the line, if we wish to edit this.